towers/clio/level_003.rb

```diff
@@ -1,15 +1,90 @@
-#  ---
-# |@ >|
-#  ---
+#  ------------------------
+# |b                  b    |
+# |    b                   |
+# |           .            |
+# |@          .        b   |
+# |           .b           |
+# |          b.           .|
+# |i                     B>|
+#  ------------------------
 
-description "TODO"
-tip "TODO"
+description "As you progress, you can't help but notice that things are changing around you. So many bugs..."
+tip "No one likes bugs. Get rid of them if you can."
+clue "The Queen is a terrific opponent. There is a way to defeat her without violence, if search deeply enough."
 
 time_bonus 20 #TODO: Refine
 ace_score 19 #TODO: Refine
-size 3, 1
-stairs 2, 0
+size 24, 7
+stairs 23, 6
 
-warrior 0, 0, :east do |u|
-  u.add_abilities :walk!, :feel, :direction_of_stairs, :talk!
+warrior 0, 3, :east
+
+unit :bug, 0, 0, :south do |u|
+  u.log_message = "Win"
+end
+unit :bug, 19, 0, :south do |u|
+  u.log_message = "Pizza is the best thing in life"
+end
+unit :bug, 4, 1, :west do |u|
+  u.log_message = "Win"
+end
+unit :bug, 20, 3, :south
```

Comment:
No log message for this bug? Does killing it work okay, or does having it be `nil` / undefined cause an error?

Reply:
Yeah, I did this on purpose, to force some critical review of the bugs (same as the bug that logs something about pizza)

Reply:
@kevin-edelmann-clio I just tested, logging with nil just writes nil in the log:


I, [2023-11-03T10:08:05.510559 #16446]  INFO -- : nil